src/processors.ts

```diff
@@ -51,7 +51,7 @@ import { camelCase } from "change-case";
 import { type TNotionUniqueId, type TDocType } from "./types/processed-data.js";
 import { AgentsPageSchema } from "./types/page-schemas/agent.js";
 
-function processScopes(pages: unknown): TProcessedScopesById {
+async function processScopes(pages: unknown): Promise<TProcessedScopesById> {
```

Comment:
why are these functions being made async and returning promises when they don't have any `await`?